components/layout/AppNavbar.vue

```diff
@@ -11,6 +11,13 @@
          <div class="flex md:space-x-62px lg:space-x-128px hidden md:block">
              <NuxtLink to="/projects">{{ $t('navbar.navbarProjects') }}</NuxtLink>
              <NuxtLink to="/dao-project-overview">{{ $t('navbar.navbarDao') }}</NuxtLink>
+             <app-link to="https://github.com/oceanprotocol/oceandao/wiki/Grant-Proposal-Template">
+                <app-button-style
+                  class="px-4 text-center"
+                  :icon="require('@/assets/images/icons/vote-white.svg')"
+                  :text="$t('project.proposal.submit')"
+                />
+             </app-link>
```

Comment:
![pearl-navbar](https://user-images.githubusercontent.com/14369479/120480497-65d3bb00-c3af-11eb-83e9-990cef08d3fd.PNG)

issue(blocking): see the attached image. there are still problems regarding to proper styling for some viewports.